1.8/administration/banning

```diff
@@ -0,0 +1,45 @@
+---
+layout: page
+title:  "Banning"
+categories: [administration]
+---
+
+The MyBB Admin CP Banning page, found under the "Configuration" tab, offers the ability to ban IP Addresses and disallow usernames and email addresses. Wildcard support is included for IP addresses, usernames, and email addresses, and for all you can view the date banned and the last attempted use. More information for each sub-page is found below.
+
+## Banned IPs
+
+Banned IP addresses disallow access for anyone accessing your MyBB forum from the IP addresses banned. Also supported is banning ranges of IP addresses, using \* to represent the wildcard. You can view when each IP address was banned, and when the IP address last attempted to access the forum.
+
+#### Banning IPs
+
+To ban an IP address, just enter the IP address, or range of IP addresses using an asterisk (\*) to represent the wildcard, into the form at the bottom of the page and select "Ban IP Address."
+
+#### Removing IP Bans
+
+You can later delete any banned IP address by simply selecting the delete icon on the right hand side of the page for the corresponding IP address.
+
+## Disallowed Usernames
+
+Disallowing usernames allows you to prevent usernames from being registered by users. From the Admin CP, you can view the date the username was disallowed as well as its last attempted use. It is important to note that disallowing a username does not affect already registered users, and does not remove any access for existing users; it only disallows the username for new registrations.
+
+#### Disallowing Usernames
+
+To ban a username, simply fill in the desired username to be banned in the form at the bottom of the page and select "Disallow Username." Be aware that partial matches are disallowed, so any username containing the disallowed username will not be able to be registered.
```

Comment:
This should read as:
 To ban a username, simply fill in the desired username to be banned in the form and select "Disallow Username." Be aware that partial matches are disallowed, so any username containing the disallowed username will not be able to be registered.